core/src/components/datetime/datetime.tsx

```diff
@@ -318,6 +318,12 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() showDefaultTimeLabel = true;
 
+  /**
+   * If `cover`, the width of the `ion-datetime` component will be set to `100%`.
+   * If unset or `fixed`, the width of the component will be set to `350px`.
```

Comment:
```suggestion
   * If `cover`, the `ion-datetime` will expand to cover the full width of its container.
   * If `fixed`, the `ion-datetime` will have a fixed width.
```

I typically prefer to write these prop comments in terms of what it does to the user's application rather than specific values. How you word it is up to you, but I would stay away from listing specific values like `100%` or `350px` (especially since those can be changed by the user)

Reply:
Totally agree with your thought here, thanks 💯